### ISSUE TITLE
where is the test/validation loss/accuracy?

### ISSUE THREAD

**Issue by lixilinx** [tensorflow/kfac#20]
I do not find any performance index on the test/validation set. The mnist.load_mnist just load the train data? It might be better to evaluate the test loss occasionally. Also, the numbers 3.5/14 (K-FAC converge >3.5x faster in >14x fewer iterations than SGD+momentum) are based on the train loss or the test loss?

**Comment by james-martens** [comment]
It's really not possible to give any particular numbers for how much faster one optimizer is than another since it depends on the context, what is your threshold of converges, etc.  I'll remove that comment from the document.
